rules/docker-dhcp-relay.mk

```diff
@@ -15,14 +15,38 @@ $(DOCKER_DHCP_RELAY)_DBG_IMAGE_PACKAGES = $($(DOCKER_CONFIG_ENGINE_BUSTER)_DBG_I
 
 $(DOCKER_DHCP_RELAY)_LOAD_DOCKERS = $(DOCKER_CONFIG_ENGINE_BUSTER)
 
-SONIC_DOCKER_IMAGES += $(DOCKER_DHCP_RELAY)
-SONIC_INSTALL_DOCKER_IMAGES += $(DOCKER_DHCP_RELAY)
+$(DOCKER_DHCP_RELAY)_VERSION = 1.0.0
+$(DOCKER_DHCP_RELAY)_PACKAGE_NAME = dhcp-relay
+$(DOCKER_DHCP_RELAY)_PACKAGE_DEPENDS = database^1.0.0
+
+$(DOCKER_DHCP_RELAY)_SERVICE_REQUIRES = updategraph
+$(DOCKER_DHCP_RELAY)_SERVICE_AFTER = swss syncd teamd
+$(DOCKER_DHCP_RELAY)_SERVICE_BEFORE = ntp-config
+$(DOCKER_DHCP_RELAY)_SERVICE_DEPENDENT_OF = swss
 
+SONIC_DOCKER_IMAGES += $(DOCKER_DHCP_RELAY)
 SONIC_DOCKER_DBG_IMAGES += $(DOCKER_DHCP_RELAY_DBG)
-SONIC_INSTALL_DOCKER_DBG_IMAGES += $(DOCKER_DHCP_RELAY_DBG)
+
+ifeq ($(INCLUDE_KUBERNETES),y)
+$(DOCKER_DHCP_RELAY)_DEFAULT_FEATURE_OWNER = kube
+endif
+
+$(DOCKER_DHCP_RELAY)_DEFAULT_FEATURE_STATE_ENABLED = y
+
+ifeq ($(INSTALL_DEBUG_TOOLS),y)
```

Comment:
Just a clarification for me, not a review comment - Should 'INSTALL_DEBUG_TOOLS' necessarily mean the debug images/dockers ?

Reply:
Yes, as you can see for regular dockers it means we will install -dbg versions if INSTALL_DEBUG_TOOLS=y
https://github.com/Azure/sonic-buildimage/blob/master/platform/vs/one-image.mk#L8

Reply:
Does this mean, default build of image (say, make target/sonic-broadcom.bin) will not build dhcp_relay and dhcp_relay docker will not be part of final install image ?